pkgs/tools/misc/conserver/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "conserver";
+  version = "8.2.7";
+
+  src = fetchFromGitHub {
+    owner = "bstansell";
+    repo = "conserver";
+    rev = "v${version}";
+    sha256 = "sha256-LiCknqitBoa8E8rNMVgp1004CwkW8G4O5XGKe4NfZI8=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  doCheck = false;
```

Comment:
Why?

Reply:
Good question: Ill dig into what happened on the tests, I actually suspect it might be due to the default build options pointed out in: https://github.com/NixOS/nixpkgs/pull/181543#pullrequestreview-1039686214

Reply:
Add my findings in https://github.com/NixOS/nixpkgs/pull/181543#issuecomment-1186350878

Add a comment about the doCheck as well